### ISSUE TITLE
[Question] Will playback sync be implemented?

### ISSUE THREAD

**Issue by nielsbauman** [iamkroot/trakt-scrobbler#194]
I have looked through the Wiki and a bit through the code as well but did not find any support for syncing watched progress _to_ the media player.
For example, if I have watched a show or movie for x% on another device and synced that to Trakt, I would like my media player to start from that point when I open the respective video.
Do you have any plans to implement such a feature?

**Comment by iamkroot** [comment]
Not really. This feature would immensely complicate the application (need to control the media player somehow), and is best left to a separate app. I don't currently need this feature, so I'm not very eager to build this either.
